linkerd/app/inbound/src/http/server.rs

```diff
@@ -72,39 +60,32 @@ impl<H> Inbound<H> {
                 // the request may have been downgraded from a HTTP/2 orig-proto request.
                 .push(http::NewNormalizeUri::layer())
                 .push(NewSetIdentityHeader::layer(()))
-                .push_on_service(
-                    svc::layers()
-                        // Downgrades the protocol if upgraded by an outbound proxy.
-                        .push(http::orig_proto::Downgrade::layer())
-                        // Limit the number of in-flight inbound requests.
-                        //
-                        // TODO(ver) This concurrency limit applies only to
-                        // requests that do not yet have responses, but ignores
-                        // streaming bodies. We should change this to an
-                        // HTTP-specific imlementation that tracks request and
-                        // response bodies.
-                        .push(svc::ConcurrencyLimitLayer::new(max_in_flight_requests))
-                        // Shed load by failing requests when the concurrency
-                        // limit is reached.
-                        .push(svc::LoadShed::layer()),
-                )
+                // Downgrades the protocol if upgraded by an outbound proxy.
+                .push_on_service(http::orig_proto::Downgrade::layer())
+                // Limit the number of in-flight inbound requests.
+                //
+                // TODO(ver) This concurrency limit applies only to
+                // requests that do not yet have responses, but ignores
+                // streaming bodies. We should change this to an
+                // HTTP-specific imlementation that tracks request and
+                // response bodies.
+                .push_on_service(svc::ConcurrencyLimitLayer::new(max_in_flight_requests))
+                // Shed load by failing requests when the concurrency
+                // limit is reached.
+                .push_on_service(svc::LoadShed::layer())
```

Comment:
out of curiosity: does separating these into multiple `push_on_service` calls also have an impact on symbol size? or, was it just a drive-by refactor?

Reply:
The additional layer types don't help anything when we're looking at the type signatures. This keeps them a little simpler. Also the added benefit of outdenting things...